### PR TITLE
Document and improve whitespace in jump_table.py

### PR DESCRIPTION
**Pull request by DanielSank**
This is purely documentation and whitespace. Changes in logic/content forthcoming.

I didn't touch the test functions since they look reasonable and because I'm trying to keep each PR smallish.

### CODE REVIEW

File: GHzDACs/jump_table.py [hunk 1]
```diff
@@ -10,8 +10,13 @@
 
 
 class JumpEntry(object):
-    """ A single entry in the jump table.
+    """A single entry in the jump table.
 
+    Attributes:
+        from_addr(int): SRAM cell at which this entry fires.
+        to_addr(int): SRAM cell to which we go after executing this
+            entry.
```
Comment (pomalley) [inline GHzDACs/jump_table.py]: "Optional; depends on operation." or something, maybe

File: GHzDACs/jump_table.py [hunk 2]
```diff
@@ -10,8 +10,13 @@
...
+        operation (Operation): Operation performed by this entry.
```
Comment (pomalley) [inline GHzDACs/jump_table.py]: operation (Operation): operation, buffalo buffalo buffalo

File: GHzDACs/jump_table.py [hunk 3]
```diff
@@ -60,15 +65,22 @@ def __str__(self):
...
 
     def as_int(self):
-        """ Return the integer equivalent to the byte code for operation.
+        """Get the integer equivalent to the byte code for operation.
 
-        It represents two (little endian) bytes. It can be translated to bytes
-        with ``util.littleEndian(my_operation.as_int(), 2)``.
+        Each operation is represented by two bytes in memory (little
+        endian). This function returns an int which will convert to the
+        right little endian bytes when passed into util.littleEndian as
```
Comment (pomalley) [inline GHzDACs/jump_table.py]: right -> correct

File: GHzDACs/jump_table.py [hunk 4]
```diff
@@ -60,15 +65,22 @@ def __str__(self):
...
+        ``util.littleEndian`(operation.as_int(), 2)``.
```
Comment (pomalley) [inline GHzDACs/jump_table.py]: extra backtick after `Endian`

File: GHzDACs/jump_table.py [hunk 5]
```diff
@@ -78,6 +90,7 @@ def __str__(self):
         return "%s %d cycles" % (self.NAME, self.cycles)
 
     def as_int(self):
+        """See Operation.as_int."""
```
Comment (pomalley) [inline GHzDACs/jump_table.py]: Might it be better to leave off the docstring if we're not going to overwrite it? I don't know.

File: GHzDACs/jump_table.py [hunk 6]
```diff
@@ -109,25 +143,35 @@ def __str__(self):
...
     def __str__(self):
         return self.NAME
 
     def as_int(self):
+        """See Operation.as_int."""
         return 5
 
 
 class CYCLE(Operation):
+    """Cycle back to anoter SRAM address.
```
Comment (pomalley) [inline GHzDACs/jump_table.py]: another

**Comment by pomalley** [comment]
LGTM, pending comments. Go ahead and merge it in when you're ready.

File: GHzDACs/jump_table.py (hunk 2 above)
Comment (DanielSank) [inline GHzDACs/jump_table.py]: Oh man, you're right. Yeesh.

File: GHzDACs/jump_table.py (hunk 5 above)
Comment (DanielSank) [inline GHzDACs/jump_table.py]: I don't know either :|

File: GHzDACs/jump_table.py (hunk 1 above)
Comment (DanielSank) [inline GHzDACs/jump_table.py]: Not sure what you mean by this. If `to_addr` is optional, shouldn't it default to `None` or something?

**Comment by DanielSank** [comment]
@pomalley I'm more than happy to add changes based on the comment about optional parameters to `JumpEntry` once I understand what you're getting at. I merged this because I want a clean slate on which to make substantive changes, and I suspect what you were pointing out slightly non-trivial because it involves making an existing argument optional.

**Pull request merged**